examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,424 @@
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 170
+    }
+   },
+   "cell_type": "code",
+   "source": [
+    "# Run and print a shell command.\n",
+    "def run(cmd):\n",
+    "  print('>> {}'.format(cmd))\n",
+    "  !{cmd}\n",
+    "  print('')\n",
+    "\n",
+    "def save_to_file(content, file_name):\n",
+    "  with open(file_name, 'w') as f:\n",
+    "    f.write(content)\n",
+    "\n",
+    "# Install apache-beam.\n",
+    "run('pip install --quiet apache-beam')\n",
+    "\n",
+    "# Copy the input file into the local file system.\n",
+    "run('mkdir -p data')\n",
+    "run('gsutil cp gs://apache-beam-samples/SMSSpamCollection/SMSSpamCollection data/SMSSpamCollection.csv')"
+   ],
+   "execution_count": null,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Inspect the data\n",
+    "Let’s see how our data looks like."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "run('head data/SMSSpamCollection.csv')\n",
+    "run('wc -l data/SMSSpamCollection.csv')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This dataset is a `csv` file with 5,574 rows and 2 columns recording the following attributes separated by a tab sign:\n",
+    "1. `Column 1`: The label (either `ham` or `spam`)\n",
+    "2. `Column 2`: The SMS as raw text (type `string`)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## First pipeline\n",
+    "We’ll start with creating a pipeline which loads the data, filters out valid messages leaving spam, and saves only valid lines to a file.\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  - type: ReadFromText\n",
+    "    name: SmsData\n",
+    "    file_pattern: data/SMSSpamCollection.csv\n",
+    "\n",
+    "  - type: PyMap\n",
+    "    name: SplitLine\n",
+    "    input: SmsData\n",
+    "    fn: 'lambda line: line.split(\"\\\\t\")'\n",
+    "\n",
+    "  - type: PyFilter\n",
+    "    name: KeepSpam\n",
+    "    input: SplitLine\n",
+    "    keep: 'lambda row: row[0] == \"spam\"'\n",
+    "\n",
+    "  - type: WriteToText\n",
+    "    name: SaveToText\n",
+    "    input: KeepSpam\n",
+    "    file_path_prefix: \"data/result-pipeline-01\"\n",
+    "    file_name_suffix: \".txt\"\n",
+    "'''\n",
+    "save_to_file(pipeline, 'pipeline-01.yaml')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this example, each transformation contains the 'input' key, but if the pipeline is linear, such as ours, we can let the inputs be implicit by designating the pipeline as a `chain` type.\n"
```

Comment:
Add a sentence here on what a `chain` type is?

Reply:
Can we remove 'chain' from this tutorial? It could be unnecessary overhead for new YAML users and we can cover in the advanced section of the documentation instead.

Reply:
I rewrote the text around the `type:chain`, please take another look.

Reply:
I would rather keep the inputs and remove "chain" entirely since it's more advanced syntax.

Reply:
Ok, I'll do that!